src/processing/dom/index.js

```diff
@@ -166,6 +148,11 @@ export default class DomProcessor {
                 selector:          selectors.IS_SVG_ELEMENT_WITH_XLINK_HREF_ATTR,
                 urlAttr:           'xlink:href',
                 elementProcessors: [this._processSVGXLinkHrefAttr, this._processUrlAttrs]
+            },
+            {
+                selector:          selectors.IS_SVG_ELEMENT_WITH_XML_BASE_ATTR,
+                urlAttr:           'xml:base',
+                elementProcessors: [this._processUrlAttrs]
             }
         ];
```

Comment:
Then `processUrlAttrs` will be called on each svg element, even if it contains two url attributes or not contains no one